### ISSUE TITLE
Sleep Inertia

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10063]
**Preferred term label:**

**Synonyms** sleep drunkenness

**Definition (free text, please give PubMed ID)**
Difficulty transitioning from sleep to wake

PMID: 36401976 Another feature sometimes associated with hypersomnolence disorders is sleep inertia (difficulty transitioning from sleep to wake), characterized by persistent sleepiness and poorly restorative sleep quality, frequent reentries into sleep, impaired performance, reduced vigilance, confusion, and irritability . Sleep inertia is reported by 26% of healthy controls [13], but a severe form of sleep inertia, sometimes called “sleep drunkenness,” does not occur in healthy controls.

PMID: 28778232 Sleep inertia/sleep drunkenness:
Patients with IH often demonstrate prolonged and pronounced difficulty with awakening from nocturnal sleep and daytime naps. This symptom is sometimes referred to as “sleep drunkenness”, to distinguish it from the milder and physiologic state of sleep inertia seen even in healthy controls .

**Parent term (use hpo.jax.org/app)** 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
- Idiopathic hypersomnia   ORPHA 33208
- Obstructive sleep apnea  OMIM 107650
- Medication or substance use/abuse
- Psychiatric disorders (most typically depression)

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000456
- label: Sleep drunkeness
- parents: [Sleep abnormality (HP:0002360)](https://hpo.jax.org/app/browse/term/HP:0002360)
- pmid: PMID:28778232|PMID:36401976
- def: Sleep drunkenness refers to the prolonged and pronounced difficulty with awakening from nocturnal sleep and daytime naps. This symptom should be distinguished from the milder and physiologic state of sleep inertia seen even in healthy controls .
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10063
- synonyms:
